Update Environment Variables documentation (#512)
anitab-org/mentorship-backend: docs/environment-variables.md

| Environment Variable     | Description                                                                                                                                                                                                                    | Example |
|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|

> | FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | dev     |

Reviewer: ```suggestion
| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | local     |
```

| Environment Variable     | Description                                                                                                                                                                                                                    | Example |
|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|

> | FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  To use the development environment configuration you should use "dev" as a value and If you want to use local database such as sqlite then set your environment config value as "local".| dev     |

Reviewer: ```suggestion
| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | dev     |
```

These are the currently available run configurations:

> - **dev:** Development environment used when developing locally

Reviewer: ```suggestion
- **dev:** Development environment used when developing with a remote database configuration
```

- **test:** Testing environment used when running tests

> - **local:** Local environment used when developing locally

Reviewer: Can you switch line 25 with 27, just to give higher attention to the local configuration

export FLASK_ENVIRONMENT_CONFIG=dev
```

> Windows user should use **set** instead of **export** for setting these environment variables. <br/>

Reviewer: Nice addition, I had no idea :)
